packages/common-all/src/dnode.ts

```diff
@@ -1372,22 +1372,34 @@ export class SchemaUtils {
         note.body = tempNote.body;
       }
 
-      // Apply date variable substitution to the body based on mustache delimiter if applicable
-      // E.g. if template has {{ CURRENT_YEAR }}, new note will contain 2021
-      // Use mustache delimiter
-      // TODO: Renable date variable substitution when new format is thought through
-      /*_.templateSettings.interpolate = /\{\{([\s\S]+?)\}\}/g;
-
+      // Apply date variable substitution to the body based on lodash interpolate delimiter if applicable
+      // E.g. if template has <%= CURRENT_YEAR %>, new note will contain 2021
       const currentDate = Time.now();
-      const compiledSchemaTemplate = _.template(note.body);
-      note.body = compiledSchemaTemplate({
-        CURRENT_YEAR: currentDate.toFormat("yyyy"),
-        CURRENT_MONTH: currentDate.toFormat("LL"),
-        CURRENT_DAY: currentDate.toFormat("dd"),
-        CURRENT_HOUR: currentDate.toFormat("HH"),
-        CURRENT_MINUTE: currentDate.toFormat("mm"),
-        CURRENT_SECOND: currentDate.toFormat("ss"),
-      });*/
+
+      note.body = note.body.replace(
+        new RegExp("<%=\\s*CURRENT_YEAR\\s*%>", "g"),
```

Comment:
Nitpick: Could we use regular expression literals? Like `/<%=\s*CURRENT_YEAR\s*%>/g`

Reply:
Sure, no problem